src/quadratic.jl

```diff
@@ -36,9 +36,9 @@ function coefficients(q::Quadratic, N, d)
     symm, sym, symp =  symbol(string("cm_",d)), symbol(string("c_",d)), symbol(string("cp_",d))
     symfx = symbol(string("fx_",d))
     quote
-        $symm = convert(TIndex, .5 * ($symfx - .5)^2)
-        $sym  = convert(TIndex, .75 - $symfx^2)
-        $symp = convert(TIndex, .5 * ($symfx + .5)^2)
+        $symm = convert(TIndex, 1//2 * ($symfx - 1//2)^2)
```

Comment:
It's a good question. Until proven otherwise :smile:, my thinking is that since
```julia
julia> promote_type(Rational{Int}, Float64)
Float64
```
we don't lose anything by specifying them as Rationals.